From 6c879fa35bf4980930e7973dc83c09da1020da56 Mon Sep 17 00:00:00 2001
From: Ido Reis <idor@ti.com>
Date: Tue, 11 Sep 2012 11:31:34 +0300
Subject: [PATCH] Remove OMAP WLAN fimrware, modify makefile

we change the wl12xx-wlan-fw-products.mk and remove TGZ references of wlan fw.
the tgz file content is untouched.

Change-Id: Idf963d03546d51428c2e9d2fd5b1299273e0df87
Signed-off-by: Ido Reis <idor@ti.com>
---
 wl12xx/wlan/wl12xx-wlan-fw-products.mk |   12 ++----------
 1 files changed, 2 insertions(+), 10 deletions(-)

diff --git a/wl12xx/wlan/wl12xx-wlan-fw-products.mk b/wl12xx/wlan/wl12xx-wlan-fw-products.mk
index 6b7671e..007349d 100644
--- a/wl12xx/wlan/wl12xx-wlan-fw-products.mk
+++ b/wl12xx/wlan/wl12xx-wlan-fw-products.mk
@@ -17,13 +17,5 @@
 # WLAN firmware package, which is defined in the top-level Android.mk
 # of the proprietary-open tree
 
-WLAN_TGZ := device/ti/proprietary-open/wl12xx/wlan/wlan.tgz
-PRODUCT_PACKAGES += \
-        wl128x-fw-4-mr.bin \
-        wl128x-fw-4-plt.bin \
-        wl128x-fw-4-sr.bin \
-        wl127x-fw-4-mr.bin \
-        wl127x-fw-4-plt.bin \
-        wl127x-fw-4-sr.bin \
-        wl1271-nvs.bin \
-        wl1271-nvs_127x.bin
+WLAN_TGZ :=
+
-- 
1.7.9

